examples/test/simple_test.cpp

```diff
@@ -0,0 +1,7 @@
+#define CATCH_CONFIG_MAIN
```

Comment:
```c++
#define CATCH_CONFIG_MAIN
``` 
This is not required in cache2-v3 or higher. Please check this document: https://github.com/catchorg/Catch2/blob/devel/docs/migrate-v2-to-v3.md

Reply:
good catch, change made